### PR TITLE
Follow user list

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: app/assets/javascripts/profile.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 $(document).on("turbolinks:load", function() {
   set_date_picker($(".input-daterange"));
   set_date_picker($(".date-picker"));
+
+  $(".btn-load-more").on("click", function() {
```
Comment (NgengChhengkim) [inline app/assets/javascripts/profile.js]: unbind click cho chac.

File: app/controllers/static_pages_controller.rb [hunk 2]
```diff
@@ -5,4 +5,24 @@ def home
     @post = Post.new
     @post.build_job_post
   end
+
+  def followings
+    @user = User.find params[:user_id]
```
Comment (NgengChhengkim) [inline app/controllers/static_pages_controller.rb]: @user de chung. load_and_autthorize_resource

Comment (NgengChhengkim) [reply app/controllers/static_pages_controller.rb]: /user/:id/following  de trong static_page nhin ko dep lam.

File: app/views/profiles/_user_media.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div class="col-md-4 user-<%= user.id %>">
+  <div class="media user-media">
+    <div class="media-left">
+      <%= link_to image_tag(user.avatar_url, class: "media-object",
+        size: Settings.images.sizes.small_avatar), user %>
+    </div>
+    <div class="media-body">
+      <div class="media-heading">
+        <strong><%= link_to user.name, user %></strong> <br>
+        <small><i><%= user.user_profile.current_position if user.user_profile %></i></small>
```
Comment (NgengChhengkim) [inline app/views/profiles/_user_media.html.erb]: delegate

File: app/views/profiles/_follower.html.erb [hunk 4]
```diff
@@ -0,0 +1 @@
+<%= render "profiles/user_media", user: follower.follower %>
```
Comment (NgengChhengkim) [inline app/views/profiles/_follower.html.erb]: sao lai follower.follower?

File: app/controllers/static_pages_controller.rb (hunk 2 above)
Comment (yongsokheng) [reply app/controllers/static_pages_controller.rb]: dak nov na cheng? :smile:

File: app/views/profiles/_follower.html.erb (hunk 4 above)
Comment (yongsokheng) [reply app/views/profiles/_follower.html.erb]: => relationship table, follower_id

File: app/controllers/static_pages_controller.rb (hunk 2 above)
Comment (NgengChhengkim) [reply app/controllers/static_pages_controller.rb]: ber ot deng dak na tao controller thmey tov

Comment (yongsokheng) [reply app/controllers/static_pages_controller.rb]: mean tae cheng hoy cheng, tha kchil tao ban dak neng mong

File: app/views/profiles/_follower.html.erb (hunk 4 above)
Comment (NgengChhengkim) [reply app/views/profiles/_follower.html.erb]: :-1:

Comment (NgengChhengkim) [reply app/views/profiles/_follower.html.erb]: relationship.follower

File: app/controllers/follow_infos_controller.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+class FollowInfosController < ApplicationController
+  before_action :load_user
```
Comment (NgengChhengkim) [inline app/controllers/follow_infos_controller.rb]: load_and_authorize_resource :user

File: app/controllers/follow_infos_controller.rb [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+  def followings
+    @followings = @user.followeds.page params[:page]
```
Comment (NgengChhengkim) [inline app/controllers/follow_infos_controller.rb]: N+1 query

File: app/controllers/follow_infos_controller.rb [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    respond_to do |format|
+      format.html {render "profiles/index"}
+      format.js
+    end
+  end
+
+  def followers
+    @followers = @user.followers.page params[:page]
```
Comment (NgengChhengkim) [inline app/controllers/follow_infos_controller.rb]: N+1 query

File: app/views/profiles/_following_list.html.erb [hunk 8]
```diff
@@ -0,0 +1,19 @@
+<%= render "profiles/tab", tab_name: "followings", user: user %>
+<div class="tab-content">
+  <div class="row following-list">
+    <%= render partial: "profiles/following", collection: followings,
```
Comment (NgengChhengkim) [inline app/views/profiles/_following_list.html.erb:4]: kon leng neng hao @followings mong ort kert te?

Comment (yongsokheng) [reply app/views/profiles/_following_list.html.erb:4]: hav @followings lus tra tae _following nov knong /followings

**Pull request merged**